src/aces/__main__.py

```diff
@@ -173,7 +173,8 @@ def main(cfg: DictConfig):
                 logger.warning("Output dataframe is empty; adding an empty patient ID column.")
                 result = result.with_columns(pl.lit(None, dtype=pl.Int64).alias("patient_id"))
                 result = result.head(0)
-
+        if cfg.window_stats_filepath:
+            result.write_parquet(cfg.window_stats_filepath)
```

Comment:
should this be run regardless of the data format? I think its fine to also decide that is just something to consider for the longer term, not locally

Reply:
I think it should only be run for meds as for the other format there is no extra processing of the raw windows, from my understanding.

Reply:
Yeah, this is fine for now, but eventually we should standardize the output format across input formats.